### ISSUE TITLE
Board Editor : Alignment & distribution tools

### ISSUE THREAD

**Issue by doom-fr** [LibrePCB/LibrePCB#595]
##### SUMMARY
To help making wonderful boards, it will be useful to have alignment and distribution tools for devices (at least).

##### SOLUTION
implement alignment & distribution tools set of Inkscape

![image](https://user-images.githubusercontent.com/7809481/69501033-36faa000-0f01-11ea-8f05-b78dd77b34fb.png)

**Comment by EdizonTN** [comment]
Yep, this will be helpfull!


> To help making wonderful boards, it will be useful to have alignment and distribution tools for devices (at least).

small correction:

To help making wonderful boards, it will be useful to have alignment and distribution tools for **selected** devices (at least).

**Comment by dbrgn** [comment]
Yep, I also like the tool provided by Inkscape. It's quite flexible.

**Comment by doom-fr** [comment]
It can be aligning devices, but also pad of devices.
